README.md

```diff
@@ -27,7 +27,7 @@ Roger is a multi-tenant, high performance job processing system for Elixir.
 
 ### 1. Check requirements
 
-- Elixir 1.5.1 or greater
+- Elixir 1.9.0 or greater
```

Comment:
This conflicts with the tool-versions, which mentions 1.11.3 as a minimum requirement